pygluu/kubernetes/terminal/volumes.py

```diff
@@ -98,13 +98,22 @@ def prompt_volumes(self):
             self.settings.set("global.azureStorageAccountType",
                               click.prompt("Please enter the volume type.", default="StandardSSD_LRS"))
 
+        elif self.settings.get("installer-settings.volumeProvisionStrategy") == "microk8sDynamic":
+                    self.settings.set("global.storageClass.provisioner", "microk8s.io/hostpath")
+
+        elif self.settings.get("installer-settings.volumeProvisionStrategy") == "minikubeDynamic":
+                    self.settings.set("global.storageClass.provisioner", "k8s.io/minikube-hostpath")
+
         elif self.settings.get("installer-settings.volumeProvisionStrategy") == "awsEbsDynamic":
             logger.info("AWS EKS Options ('gp2', 'io1', `io2`, 'st1', 'sc1')")
             self.settings.set("global.awsStorageType",
                               click.prompt("Please enter the volume type.", default="io1"))
+
         elif self.settings.get("installer-settings.volumeProvisionStrategy") == "gkePdDynamic":
             logger.info("GCE GKE Options ('pd-standard', 'pd-ssd')")
             self.settings.set("global.gcePdStorageType",
                               click.prompt("Please enter the volume type.", default="pd-ssd"))
-        elif "OpenEbsHostPathDynamic" in self.settings.get("installer-settings.volumeProvisionStrategy"):
```

Comment:
This was right. You are breaking a part of the installation here. Read what this statement says. There are more than one option with `OpenEbsHostPathDynamic`